drivers/sensor/hts221/Kconfig

```diff
@@ -16,23 +16,22 @@ config HTS221_NAME
 	string
 	prompt "Driver name"
 	default "HTS221"
-	depends on HTS221
+	depends on HTS221 && !HAS_DTS_SENSOR
 	help
 	  Device name with which the HTS221 sensor is identified.
```

Comment:
I'd like to update these help texts, but they're not part of your patch. Should I make a note and come back later to fix them?  For example, this would read better as:

     Device name identifying the HTS221 sensor.

Reply:
I had a look and it seems basically all sensors require an update.
If possible I'd like to avoid doing it inside this PR :-)